target/linux/ramips/dts/FON2601.dts

```diff
@@ -0,0 +1,167 @@
+// SPDX-License-Identifier: GPL-2.0
```

Comment:
From what I know, upstream prefers ```GPL-2.0 OR MIT```, ```GPL-2.0-or-later OR MIT``` .

That said, it's totally fine if you keep it as is.

Reply:
Please let me keep it.

Reply:
Ok. GPL-2.0 it is.